src/Storage/Storage.Management.Test/ScenarioTests/StorageFileTests.ps1

```diff
@@ -99,5 +99,112 @@ function Test-StorageFileShare
     }
 }
 
+<#
+.SYNOPSIS
+Test Storage File Share Soft Delete
+.DESCRIPTION
+SmokeTest
+#>
+function Test-ShareSoftDelete
+{
+    # Setup
+    $rgname = Get-StorageManagementTestResourceName;
+
+    try
+    {
+        # Test
+        $stoname = 'sto' + $rgname;
+        $stotype = 'Standard_LRS';
+        $loc = Get-ProviderLocation ResourceManagement;
+        $kind = 'StorageV2'
+		$shareName1 = "share1"+ $rgname
+		$shareName2 = "share2"+ $rgname
+
+        Write-Verbose "RGName: $rgname | Loc: $loc"
+        New-AzResourceGroup -Name $rgname -Location $loc;
+		
+        $loc = Get-ProviderLocation_Stage ResourceManagement;
+        New-AzStorageAccount -ResourceGroupName $rgname -Name $stoname -Location $loc -Type $stotype -Kind $kind 
+        $stos = Get-AzStorageAccount -ResourceGroupName $rgname;
+
+		# Enable Share Soft delete
+		Update-AzStorageFileServiceProperty -ResourceGroupName $rgname -StorageAccountName $stoname -EnableShareDeleteRetentionPolicy $true -ShareRetentionDays 5 
+		$servicePropertie = Get-AzStorageFileServiceProperty -ResourceGroupName $rgname -StorageAccountName $stoname 
+		Assert-AreEqual $true $servicePropertie.ShareDeleteRetentionPolicy.Enabled
+		Assert-AreEqual 5 $servicePropertie.ShareDeleteRetentionPolicy.Days
+
+		#create Shares
+		New-AzRmStorageShare -ResourceGroupName $rgname -StorageAccountName $stoname -Name $shareName1
+		$share = Get-AzRmStorageShare -ResourceGroupName $rgname -StorageAccountName $stoname -Name $shareName1
+		Assert-AreEqual $rgname $share.ResourceGroupName
+		Assert-AreEqual $stoname $share.StorageAccountName
+		Assert-AreEqual $shareName1 $share.Name
+		New-AzRmStorageShare -ResourceGroupName $rgname -StorageAccountName $stoname -Name $shareName2
+		
+		# Get share usage
+		$share = Get-AzRmStorageShare -ResourceGroupName $rgname -StorageAccountName $stoname -Name $shareName1 -GetShareUsage
```

Comment:
A parameter cannot be found that matches parameter name `GetShareUsage`

Reply:
Fixed. 
Plan to add -GetShareUsage in the PR, but remove since server bug. 
Product code is removed, but forget to remove test code.